src/Microsoft.WindowsAzure.Mobile.Files.Shared/File.cs

```
@@ -29,14 +26,16 @@ public static Task<Stream> OpenReadAsync(string targetPath)
 #elif WIN_APPS
         public async static Task<Stream> CreateAsync(string targetPath)
         {
-            StorageFile file = await StorageFile.GetFileFromPathAsync(targetPath);
-
-            return await file.OpenStreamForWriteAsync();
+            StorageFolder storageFolder = ApplicationData.Current.LocalFolder;
+            StorageFile file = await storageFolder.CreateFileAsync(targetPath, CreationCollisionOption.ReplaceExisting);
```

WinRT only allows creating files in Application's local folder, roaming folder or temporary folder using Storage folder's API: https://msdn.microsoft.com/en-us/windows/uwp/files/quickstart-reading-and-writing-files. Android,  only allows creating files in App's personal folder: Environment.SpecialFolder.Personal. In this case, we assume targetPath is valid for that platform and let the platform throw if the path is invalid.  For WinRT, we can either default to LocalFolder or give the option to user where to download the files.